<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModality::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>895</width>
    <height>867</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>100</horstretch>
    <verstretch>100</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>100</horstretch>
     <verstretch>100</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QGridLayout" name="buttons_layout" rowstretch="0">
        <property name="sizeConstraint">
         <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
        </property>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="tutoria_button">
            <property name="text">
             <string>Útmutató</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="load_button">
            <property name="text">
             <string>Load maps</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="sim_button">
            <property name="text">
             <string>Simulation mode</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="exit_button">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>100</horstretch>
              <verstretch>100</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Exit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="restart_button">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>100</horstretch>
              <verstretch>100</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Restart</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QScrollBar" name="Delay_input_data">
            <property name="layoutDirection">
             <enum>Qt::LayoutDirection::RightToLeft</enum>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="invertedAppearance">
             <bool>true</bool>
            </property>
            <property name="invertedControls">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="Delay_text">
            <property name="text">
             <string>Delay:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="Delay_mode">
            <property name="text">
             <string>No delay mode</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="Ex_rand_walk">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>100</horstretch>
              <verstretch>100</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Random walk</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="Ex_BFS">
            <property name="text">
             <string>BFS</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="Ex_DFS">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>100</horstretch>
              <verstretch>100</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>DFS</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="Ex_rand_walkmem">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>100</horstretch>
              <verstretch>100</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Greedy BFS</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="Ex_A_star">
            <property name="text">
             <string>A* algorithm</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="explore_button">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>100</horstretch>
              <verstretch>100</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Explore</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="Pf_BFS">
            <property name="text">
             <string>BFS</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="Pf_DFS">
            <property name="text">
             <string>DFS</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pathfind_button">
            <property name="text">
             <string>Find path</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="Obstacle_text">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>100</horstretch>
              <verstretch>100</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Number of obstacles:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="obstacle_input_data">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>100</horstretch>
              <verstretch>100</verstretch>
             </sizepolicy>
            </property>
            <property name="maximum">
             <number>398</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Obstacle_generate_button">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>100</horstretch>
              <verstretch>100</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Generate</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLabel" name="explore_text">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>100</horstretch>
              <verstretch>100</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Explore the graph:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="output_statistics">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>100</horstretch>
              <verstretch>100</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>895</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>100</horstretch>
     <verstretch>100</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
